GetIndexDefinition/function.json

```diff
@@ -3,7 +3,8 @@
     {
       "name": "indexName",
       "type": "activityTrigger",
-      "direction": "in"
+      "direction": "in",
+      "authLevel": "function"
```

Comment:
I was expecting authentication to be required for the httpTrigger only and not (as is the case here) the activities or orchestrators.

AFAIK there is no way to call an orchestration/activity directly.

Did you discover differently?

Reply:
Yeah good point. I wasn't thinking. I've removed it from all functions except `HttpStart`.